gluten-core/src/main/scala/io/glutenproject/expression/BinaryExpressionTransformer.scala

```diff
@@ -36,6 +36,13 @@ class BinaryExpressionTransformer(
     original: Expression)
   extends ExpressionTransformer with Logging {
   override def doTransform(args: java.lang.Object): ExpressionNode = {
+    if (ExpressionMappings.REPEAT.equalsIgnoreCase(original.prettyName) &&
+      !BackendsApiManager.getBackendName.equalsIgnoreCase(GlutenConfig.GLUTEN_CLICKHOUSE_BACKEND)) {
+      val errMsg = String.format("%s or %s is not currently supported for non clickhouse backend.",
+        original.getClass, original)
+      logWarning(errMsg)
+      throw new UnsupportedOperationException(errMsg)
```

Comment:
How about adding this function into below blacklist for Velox backend to save the extra check here?
[VeloxExpressionUtil.scala#L31](https://github.com/oap-project/gluten/blob/main/backends-velox/src/main/scala/io/glutenproject/utils/VeloxExpressionUtil.scala#L31)

Reply:
OK